src/OpenTK.Windowing.GraphicsLibraryFramework/Input/MouseState.cs

```diff
@@ -175,6 +181,32 @@ public bool WasButtonDown(MouseButton button)
             return _buttonsPrevious[(int)button];
         }
 
+        /// <summary>
+        /// Gets whether the specified mouse button is pressed in the current frame but released in the previous frame.
```

Comment:
Let's make this really clear, because down vs pressed is always a point of confusion.
"A button is pressed only on the first frame that it is held, after which, it will still be down, but not pressed."